diff --git a/lib/cmake/duckdb.cmake b/lib/cmake/duckdb.cmake
index 60aa9803..20cc2017 100644
--- a/lib/cmake/duckdb.cmake
+++ b/lib/cmake/duckdb.cmake
@@ -82,6 +82,7 @@ target_link_libraries(
   INTERFACE ${install_dir}/lib/libduckdb_pg_query.a
   INTERFACE ${install_dir}/lib/libduckdb_utf8proc.a
   INTERFACE ${install_dir}/lib/libduckdb_fastpforlib.a
+  INTERFACE ${install_dir}/lib/libicu.a
   INTERFACE dl)
 
 target_include_directories(
